src/tests/unit_x509.cpp

```diff
@@ -1592,8 +1593,885 @@ Test::Result test_x509_tn_auth_list_extension_decode() {
    return result;
 }
 
+Test::Result test_x509_ip_addr_blocks_extension_decode() {
```

Comment:
Thank you for the careful tests. This file is already growing quite large; please move all of these tests into a new file eg `test_x509_rpki.cpp` and add them as a new `Test`

[I don't know if `rpki` is the best term to use here, if you think something else is better feel free to use that]